app-design-center/enterprise-production/README.md

```diff
@@ -0,0 +1,3 @@
+# Enterprise Production Helm Chart
+
+These samples show how to deploy a stateless web application for production. Visit https://cloud.google.com/kubernetes-engine/docs/ to follow the tutorial.
```

Comment:
Question: Should we link to https://cloud.google.com/application-design-center/docs/build-on-google-templates instead?
```suggestion
These samples show how to deploy a stateless web application for production. Visit https://cloud.google.com/application-design-center/docs/build-on-google-templates for instructions.
```